### PR TITLE
Initial changes

### PR DESCRIPTION
**Pull request by jlentzfl**
- updates to readme
- updates to job
- replace container (always cortex) with name
- update label name

With these changes, all but 2 of the Compactor and Compactor Resources dashboards have data.

### CODE REVIEW

File: cortex-mixin/dashboards/dashboard-utils.libsonnet [hunk 1]
```diff
@@ -480,6 +480,9 @@ local utils = import 'mixin-utils/utils.libsonnet';
             container_fs_writes_bytes_total{
               %s,
               container="%s",
+              //Below attempt did not work
```
Comment (tmendenhall) [inline cortex-mixin/dashboards/dashboard-utils.libsonnet:483]: do you want to leave in the comments?

**Review by tmendenhall — APPROVED** [review]
looks good. hopefully we don't have to rebuild the charts very often... with go

File: cortex-mixin/dashboards/dashboard-utils.libsonnet (hunk 1 above)
Comment (jlentzfl) [reply cortex-mixin/dashboards/dashboard-utils.libsonnet:483]: Yes, until I figure out what fixes it.

Comment (tmendenhall) [reply cortex-mixin/dashboards/dashboard-utils.libsonnet:483]: 👍

**Comment by jlentzfl** [comment]
Thanks Tyler!  Once we get them all working I wouldn't expect to have to rebuild them again unless we redeploy Cortex with a new name...

**Pull request merged**